packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -1331,6 +1368,14 @@ - (void)setUpCaptureSessionForAudio {
   // Setup the audio output.
   _audioOutput = [[AVCaptureAudioDataOutput alloc] init];
 
+  dispatch_sync(dispatch_get_main_queue(), ^{
+    upgradeAudioSessionCategory(AVAudioSessionCategoryPlayAndRecord,
+                                AVAudioSessionCategoryOptionDefaultToSpeaker |
+                                    AVAudioSessionCategoryOptionAllowBluetoothA2DP |
+                                    AVAudioSessionCategoryOptionAllowAirPlay,
+                                0);
```

Comment:
I still think it's better not to pass this param. It's making the function a bit complicated to read.

Reply:
Or you mean that better would be to have it as an objc selector rather than C function?

Reply:
I don't think it's possible to share video player and camera code. Our plugins are not setup to do so. 

I meant it seems we only pass `0` and never other values to this param. So do we really need this param?